backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/pporder/api/impl/AbstractPPOrderReceiptHUProducer.java

```diff
@@ -384,7 +385,12 @@ private IAllocationRequest createAllocationRequest(final IHUContext huContext, f
 		if (clearanceStatus != null)
 		{
 			final String language = getOrgUserOrLoggedInUSerLanguage(referencedModel);
-			clearanceStatusInfo = ClearanceStatusInfo.of(clearanceStatus, msgBL.getMsg(language, MESSAGE_ClearanceStatusInfo_Manufactured));
+			clearanceStatusInfo = ClearanceStatusInfo.builder()
+					.clearanceStatus(clearanceStatus)
+					.clearanceNote(msgBL.getMsg(language, MESSAGE_ClearanceStatusInfo_Manufactured))
+					.clearanceDate(TimeUtil.asInstant(date))
+					.build();
```

Comment:
add the org-id (or the new InstanceAndOrg-thingie?)
so that we don't end up in the wrong timezone